app/views/items/show.html.erb

```diff
@@ -29,3 +29,54 @@
     </div>
   </div>
 </div>
+
+<div class="row">
+<% if current_user %>
+    <div class="col s2"><p></p></div>
+    <div class="col s8">
+      <%= render partial: '/reviews/form' %>
+    </div>
+<% else %>
+  <div class="col s2"><p></p></div>
+  <div class="col s8">
+    <h5 id='guest-message'>Login or Create Account to Write a Review</h5>
+    <div class="row guest-links">
+        <%= button_to "Login", login_path, method: :get, class: "waves-effect waves-light btn login-to-review" %>
+        <%= button_to "Create Account", new_user_path, method: :get, class: "waves-effect waves-light btn create-acct-to-review"%>
+    </div>
+  </div>
+<% end %>
+</div>
+
+<% unless @reviews.empty? && @item.reviews.empty?%>
```

Comment:
Could this be a method in the item model? Basically a method that executes this logic and returns what is needed for the view depending on which conditional it meets?